### PR TITLE
reference-docs: add `supabase sso` CLI reference docs

### PR DESCRIPTION
**Pull request by hf**
And cross-link to the SSO guide.

### CODE REVIEW

**Comment by silentworks** [comment]
Direct link to the preview for anyone reviewing https://docs-git-hf-add-sso-docs-take-2-supabase.vercel.app/docs/guides/auth/sso/auth-sso-saml

File: apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx [hunk 1]
```diff
@@ -129,7 +131,7 @@ CREATE POLICY "View organization settings."
 
 ## Managing SAML 2.0 connections
 
-Once you've enabled SAML 2.0 support on your project via the [Auth Providers](https://app.supabase.com/project/_/auth/providers) page in the dashboard, you can use the [Supabase CLI](/docs/guides/cli) to add, update, remove and view information about identity providers.
+Once you've enabled SAML 2.0 support on your project via the [Auth Providers](https://app.supabase.com/project/_/auth/providers) page in the dashboard, you can use the [Supabase CLI](/docs/referece/cli/supabase-sso) to add, update, remove and view information about identity providers.
```
Comment (J0) [inline apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx:134]: Might be a typo, do double check

```suggestion
Once you've enabled SAML 2.0 support on your project via the [Auth Providers](https://app.supabase.com/project/_/auth/providers) page in the dashboard, you can use the [Supabase CLI](/docs/reference/cli/supabase-sso) to add, update, remove and view information about identity providers.
```

File: spec/cli_v1_commands.yaml [hunk 2]
```diff
@@ -45,6 +45,256 @@ flags:
...
+        description: |-
+          A URL (typically HTTPS) that points to a SAML 2.0 Metadata XML document describing the SAML features supported by the identity provider being added.
+          Mutually exclusive with --metadata-file.
+        type: string
+      - id: metadata-file
+        name: --metadata-file <path to file>
+        description: |
+          A path to a file containing a SAML 2.0 Metadata XML document describing the SAML features supported by the identity provider being added. Prefer using --metadata-url if possible.
+
+          Mutually exclusive with --metadata-file.
```
Comment (J0) [inline spec/cli_v1_commands.yaml:241]:
```suggestion
          Mutually exclusive with --metadata-url.
```

File: spec/cli_v1_commands.yaml [hunk 3]
```diff
@@ -45,6 +45,256 @@ flags:
...
+        description: |-
+          Removes these email domains from the existing set of domains. See --domains for a detailed explanation.
+      - id: attribute-mapping-file
+        name: attribute-mapping-file
+        type: string
+        description: |-
+          A file containing a JSON document describing the attribute mappings attached to this identity provider. User information contained in the SAML assertion will be mapped according to the rules laid out in this document.
+  - id: supabase-sso-list
+    title: supabase sso list
+    summary: Add a connection to a SSO identity provider
```
Comment (J0) [inline spec/cli_v1_commands.yaml:263]: Should the description be about listing connections instead?
> Add a connection to a SSO identity provider

File: spec/cli_v1_commands.yaml [hunk 4]
```diff
@@ -45,6 +45,256 @@ flags:
...
+    description: |-
```
Comment (J0) [inline spec/cli_v1_commands.yaml:264]: ditto

File: spec/cli_v1_commands.yaml [hunk 5]
```diff
@@ -45,6 +45,256 @@ flags:
...
+      Add and configure a new connection to a SSO identity provider to your Supabase project.
+    tags:
+      - management-api
+    links: []
+    flags: []
+  - id: supabase-sso-show
+    title: supabase sso show <provider-id>
+    summary: Show information about an identity provider.
+    description: |-
+      Provides the information about an established connection to an identity provider. You can use --output metadata to obtain the raw SAML 2.0 Metadata XML document stored in your project's configuration.
```
Comment (J0) [inline spec/cli_v1_commands.yaml:274]: Do double check this, can't seem to find the `--output metadata` option on v1.49.4 (latest version on brew)

CleanShot 2023-04-06 at 20 47 33@2x

Comment (J0) [inline spec/cli_v1_commands.yaml:274]:
```suggestion
      Provides the information about an established connection to an identity provider. You can use --output --metadata to obtain the raw SAML 2.0 Metadata XML document stored in your project's configuration.
```

File: spec/cli_v1_commands.yaml [hunk 6]
```diff
@@ -45,6 +45,256 @@ flags:
...
+        name: Show information
+        code: |-
+          supabase sso show 6df4d73f-bf21-405f-a084-b11adf19fea5 \
+            --project-ref abcdefghijklmnopqrst
+        response: |-
+          Information about the identity provider in pretty output.
+      - id: metadata-output
+        name: Get raw SAML 2.0 Metadata XML
+        code: |-
+          supabase sso show 6df4d73f-bf21-405f-a084-b11adf19fea5 \
```
Comment (J0) [inline spec/cli_v1_commands.yaml:288]: Same here for `--metadata` point - not sure if the `--` matters

File: spec/cli_v1_commands.yaml (hunk 3 above)
Comment (J0) [inline spec/cli_v1_commands.yaml:263]:
```suggestion
    summary: List all identity providers
```

File: spec/cli_v1_commands.yaml (hunk 5 above)
Comment (hf) [reply spec/cli_v1_commands.yaml:274]: Ah yes got a bit confused. Will fix the whole thing.

**Review by J0 — APPROVED** [review]
LGTM

**Pull request merged**